index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile First Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <style> @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); </style>
+  <style> @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); </style>
+  <style> @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); </style>
+</head>
+<body>
+  <header>
+    <div class="toolbar">
+      <a href="https://github.com/bilalrajput09" class="name">
+        Bilal Ahmed
+      </a>
+        <img src="C:\Users\HP\My-Portfolio\img\threelines.png">
+    </div>  
+  </header>
+  <div class="introduction">
+    <h1>Hey There.
+      I’m Bilal Ahmed</h1>
+      <h2>I’am a Software Developer</h2>
+      <h3>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</h3>
+  </div>
+  <div class="social">
+    <ul class="accounts">
+      <li>
+        <a href="https://github.com/bilalrajput09"><img src="C:\Users\HP\My-Portfolio\img\github.png">
+        </a>
+      </li>
+      <li>
+        <a href="https://twitter.com/bilalrajput09"><img src="C:\Users\HP\My-Portfolio\img\twitter.png">
+        </a>
+      </li>
+      <li>
+        <a href=https://medium.com/@ranabilalrajput09><img src="C:\Users\HP\My-Portfolio\img\med.png">
+        </a>
+      </li>
+      <li>
+        <a href="https://linkdin.com/bilalrajput09"><img src="C:\Users\HP\My-Portfolio\img\linkdin.png">
+        </a>
+      </li>
```

Comment:
- [ ] Great job 😃 at adding the images of the social media, but I see that images not appearing in the demo👌, so kindly fix this error maybe you put the path of the images wrongly☺️, so please put the correct path to let it work.

| Figma design | Your design| 
| --- | --- |
|    ![image](https://user-images.githubusercontent.com/83360636/212329283-a8d5a72a-9669-48a9-be9c-5ada1d108d62.png) |    ![image](https://user-images.githubusercontent.com/83360636/212329180-c8f72ce4-2717-4e97-be06-7cf18716bdb0.png) |